add missing docs for org_info (#13972)
fleetdm/fleet: docs/Configuration/configuration-files/README.md

The URL of the logo of the organization.

This logo is displayed in the top bar and other areas of Fleet that have dark backgrounds.

- Optional setting (string)
- Default value: none (uses Fleet's logo)
- Config file format:
  ```yaml
  org_info:
    org_logo_url: https://example.com/logo.png
  ```

##### org_info.org_logo_url_light_background

The URL of a logo of the organization that can be used with light backgrounds.

> Note: this URL is currently only used for the dialogs displayed during MDM migration

- Optional setting (string)
- Default value: none (uses Fleet's logo)
- Config file format:
  ```yaml
  org_info:
    org_logo_url_light_background: https://example.com/logo-light.png
  ```

##### org_info.contact_url

> An URL that can be used by ends user to contact the organization.

Reviewer: ```suggestion
A URL that can be used by end users to contact the organization.
```

Author: thanks!
